utils/dockerd/git-short-commit.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+#
+#	USAGE: git-short-commit.sh <GIT_URL> <GIT_REF> <GIT_DIR>
+#
+
+set -e
+
+error() {
+	echo "ERROR: ${*}" >&2
+	exit 1
+}
+
+GIT_URL="${1}"
+if [ ! -n "${GIT_URL}" ]; then
+	error "Git URL not specified"
+fi
+
+GIT_REF="${2}"
+if [ ! -n "${GIT_REF}" ]; then
```

Comment:
! -n should be -z . Run through shellcheck.

Reply:
Nice catch.
I guess that specific check isn't in version 0.5.0 of shellcheck.  
Is there a recommended version I should use?

Reply:
latest one you can I guess.

Reply:
if you're running debian 10, run

sudo apt -t buster-backports install shellcheck